Sources/NetworkService/NetworkError.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+/// Contains network errors that may occur, while the application is running.
+public enum NetworkError: Error, CaseIterable {
```

Comment:
typo:
You have two identical files in different folders

Reply:
My bad.  forgot to commit it. Thanks for you attention! 
Fixed.